src/utils/NotificationUtils.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
+import * as admin from 'firebase-admin';
+// eslint-disable-next-line import/no-unresolved
+import { getMessaging } from 'firebase-admin/messaging';
+
+const schedule = require('node-schedule');
+
+function sendNotification(deviceToken: string, notifData) {
+  const message = {
+    data: notifData,
+    token: deviceToken,
+  };
+  if (deviceToken !== '') {
+    getMessaging().send(message)
+      .then((response) => {
+        console.log(response);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+}
+
+function waitForDeparture(deviceToken: string, startTime: string) {
+  const startDate = new Date((parseInt(startTime) * 1000) - (60000 * 10));
+
+  const notifData = {
+    data: 'You should board your bus in 10 minutes',
+    notification: 'testBody',
```

Comment:
Does testBody refer to the title of the notification itself? Cuz maybe we could change it to Boarding Reminder or something like that idk